ngraph/frontend/paddlepaddle/src/place.cpp

```diff
@@ -83,20 +83,23 @@ std::vector<Place::Ptr> TensorPlacePDPD::getConsumingPorts() const
         }
         else
         {
-            PDPD_THROW("Consuming Port has expired.");
+            PDPD_CHECK(
+                ngraph::frontend::ErrorCode::ERROR_GENERAL, false, "Consuming Port has expired.");
```

Comment:
Can we make it cleaner? Add PDPD_THROW or something

Reply:
done